lib/manager/poetry/artifacts.spec.ts

```diff
@@ -114,6 +115,26 @@ describe('manager/poetry/artifacts', () => {
     expect(hostRules.find.mock.calls).toHaveLength(3);
     expect(execSnapshots).toMatchSnapshot();
   });
+  it('prioritizes pypi-scoped credentials', async () => {
+    fs.readFile.mockResolvedValueOnce(null);
+    fs.readFile.mockResolvedValueOnce('[metadata]\n' as never);
```

Comment:
It was "necessary" because readFile wants a `Buffer`. Passing the correct type now: https://github.com/renovatebot/renovate/pull/14255/commits/4bcc32edbb0755729db0e002a4210ac91c135e13